controllers/templatesControllers.js

```diff
@@ -1,34 +1,51 @@
-const {events, templateEvents, templatesList, findTemplateByID} = require(`../services/templates`)
+const templatesService = require(`../services/templates`)
 //const path = require(`node:path`);
 const {URL} = require(`url`);
+const {getNumOfSentEmails} = require("../DAL/emailsDAL");
 
-
+// ************* HTTP way of handling id passed in query string ******************/
 getTemplateID = (req) => {
     return new URL(req.url, `http://${req.headers.host}`)
         .searchParams.get('id');
 }
 
+// ************* Express way of handling Id passed in path params  ****************/
+// getTemplateID = (req) => {
+//     return  req.params.id;
+// }
+
+
+
 function getAllTemplates(req, res) {
     res.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
DRY
consider adding some middleware or handler that enrich the response with the desired headers